### PR TITLE
Karam dosa

### PR DESCRIPTION
**Pull request by SaiPraveen912**

### CODE REVIEW

**Comment by spraveen912** [comment]
Why extra oil for karam dosa?

**Comment by SaiPraveen912** [comment]
> Why extra oil for karam dosa?

For extra taste extra oil.

File: menu.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 dosa
-karam
\ No newline at end of file
+karam
+oil
```
Comment (spraveen912) [inline menu.txt:3]: Why extra oil?

Comment (SaiPraveen912) [reply menu.txt:3]: For extra taste.

**Review by spraveen912 — APPROVED** [review]
Everything is good, you can go ahead.

**Pull request merged**